ietf-network-inventory.yang

```diff
@@ -413,6 +506,34 @@ module ietf-network-inventory {
                 "This container contains some attributes belong to
                 port only.";
             }
+            container transceivers-module-specific-info {
+              when "derived-from-or-self(../class, 
+                    'nwi:transceivers-module')";
+              config false;
+              description
+                "This container contains some attributes belong to
+                transceivers modules only.";
+              container channels {
+                presence
+                  "When present it represents a multi-channel port
+                   which supports port breakouts.";
+                description
+                  "Top level container for the list of channels
+                   supported by the transceivers module.";
+                list channel {
```

Comment:
Same question as above : what are these channels? This is the OpenConfig extract to define the case a device is able to support "breakout": "When a device has the capability to break a port into interfaces of different speeds and different number of physical channels,".
My question is if the channles in your proposal is the same channel defined here.